core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NoticeContainer.java

```diff
@@ -55,6 +56,49 @@
   private final List<SystemError> systemErrors = new ArrayList<>();
   private boolean hasValidationErrors = false;
 
+  /**
```

Comment:
Please do not move large blocks of code in CLs that change logic. That hides your changes because it looks like you removed one function completely and added a new one.